src/main/cpp/subsystems/DooHickey.cpp

```diff
@@ -131,6 +131,98 @@ void DooHickey::Periodic() {
       IsRotationControlFinished = true;
     }
 
+    struct colorstuff 
```

Comment:
Let's spend some time on this on Monday (or when we have the shooter working). I have some recommendations for how to make some changes so that this is easier to use. We also need to move this to a command instead of in periodic.